kong/pdk/private/phases.lua

```diff
@@ -71,7 +71,7 @@ local function check_phase(accepted_phases)
       -- treat custom content blocks as the Admin API
       current_phase = PHASES.admin_api
     else
-      error("no phase in kong.ctx.core.phase")
+      error("no phase " .. current_phase .. "in kong.ctx.core.phase")
```

Comment:
this is inside `if not current_phase`

Reply:
hum, it's the only change in that file; probably an artifact of the rebase.  will remove